flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/expressions/TemporalTypesTest.scala

```diff
@@ -1140,41 +1140,41 @@ class TemporalTypesTest extends ExpressionTestBase {
     tableConfig.setLocalTimeZone(ZoneId.of("Asia/Shanghai"))
 
     // INT -> TIMESTAMP_LTZ
-    testAllApis(toTimestampLtz(100, 0), "TO_TIMESTAMP_LTZ(100, 0)", "1970-01-01 08:01:40.000")
```

Comment:
Converting the tests to expected behavior.

Reply:
what is the Flink behavior before this PR and after?

For instance if we run a query from flink sql client?

Reply:
Synced with Sergey, this has always been the behavior for Flink. Therefore, I will revert these changes and make my new function align with this existing behavior.